pytorch_pfn_extras/handler/_logic.py

```diff
@@ -464,3 +465,80 @@ def eval_step(
         model = models[self.model_name]
         outs = forward(model)(batch)
         return outs
+
+
+@dataclasses.dataclass
+class ClousureModelOutput:
+    outs: Any
+    loss: torch.Tensor
+
+    def __float__(self) -> float:
+        return float(self.loss)
+
+
+class ClousureLogic(Logic):
+
+    def consume_options(self, options: Dict[str, Any]) -> None:
+        super().consume_options(options)
```

Comment:
Can this function be deleted too? :)

Reply:
I can't remove this method completely because it changes the process when grad_scaler options are passed.

Most of it is the same as Logic.cosnsume_options, so calling it from the super class is almost sufficient, so most of it can be removed. We will modify this one.

Reply:
https://github.com/pfnet/pytorch-pfn-extras/issues/636#issuecomment-1493696719

We have confirmed that grad_scaler and clousure step cannot be used at the same time.